samples/Location/audit-allowed-locations/README.md

```diff
@@ -0,0 +1,31 @@
+# Audit for approved datacentre locations
```

Comment:
Lets name this policy "Audit allowed locations" to make it clear it is the audit variant of the "Allowed locations" policy

Reply:
this impacts the readme as well as the policy displayName in azurepolicy.json